Tesla/Tesla/View/UnlockView.swift

```diff
@@ -0,0 +1,150 @@
+// UnlockView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Экран блокировки
+struct UnlockView: View {
+    // MARK: - Public Properties
+
+    var body: some View {
+        backGroundView {
+            VStack {
+                HStack {
+                    Spacer()
+                    settingsButton
+                        .padding(.horizontal, 30)
+                }
+                Spacer()
+                welcomeTextView
+                Spacer()
+                carImageView
+                    .padding(.bottom, 125)
+                lockCarControlView
+                Spacer()
+            }
+        }
+        .onDisappear {
+            unlockViewModel.isUnclocked = false
+        }
+    }
+
+    // MARK: - Private Properties
+
+    @StateObject private var unlockViewModel = UnlockViewModel()
+
+    private var welcomeTextView: some View {
+        VStack {
+            Text("Hi, Ilentiy")
+            Text("Welcome back".uppercased())
+                .font(.largeTitle)
+                .bold()
+        }
+        .opacity(unlockViewModel.isUnclocked ? 1 : 0)
+    }
+
+    private var settingsButton: some View {
+        NavigationLink {
+            MenuView()
+                .navigationBarBackButtonHidden(true)
+        } label: {
+            Button(action: {}, label: {
+                Image(systemName: "gearshape.fill")
+                    .foregroundColor(Color("label"))
+                    .frame(width: 25, height: 25)
+            })
+            .opacity(unlockViewModel.isUnclocked ? 1 : 0.6)
+            .buttonStyle(NavigationButtonStyle())
+            .disabled(unlockViewModel.isUnclocked)
+        }
+    }
+
+    private var carImageView: some View {
+        Image(unlockViewModel.isUnclocked ? "teslaUnlocked" : "teslaLocked")
+            .resizable()
+            .scaledToFill()
+            .frame(height: 255)
+            .offset(x: unlockViewModel.isUnclocked ? 0 : 10)
+    }
+
+    private var lockCarControlView: some View {
```

Comment:
```suggestion
    private var lockCarControlView: some View {
```
Простынь - раздроби это вью